src/ratbag_emu/protocol/hidpp20.py

```diff
@@ -186,3 +184,165 @@ def IFeatureSet(self, data, ase, args):
+                0x09, 0x02,                    #  Usage (Vendor Usage 2)             185
+                0x91, 0x00,                    #  Output (Data,Arr,Abs)              187
+                0xc0,                          # End Collection                      189
+                0x06, 0x00, 0xff,              # Usage Page (Vendor Defined Page 1)  190
+                0x09, 0x04,                    # Usage (Vendor Usage 0x04)           193
+                0xa1, 0x01,                    # Collection (Application)            195
+                0x85, 0x20,                    #  Report ID (32)                     197
+                0x75, 0x08,                    #  Report Size (8)                    199
+                0x95, 0x0e,                    #  Report Count (14)                  201
+                0x15, 0x00,                    #  Logical Minimum (0)                203
+                0x26, 0xff, 0x00,              #  Logical Maximum (255)              205
+                0x09, 0x41,                    #  Usage (Vendor Usage 0x41)          208
+                0x81, 0x00,                    #  Input (Data,Arr,Abs)               210
+                0x09, 0x41,                    #  Usage (Vendor Usage 0x41)          212
+                0x91, 0x00,                    #  Output (Data,Arr,Abs)              214
+                0x85, 0x21,                    #  Report ID (33)                     216
+                0x95, 0x1f,                    #  Report Count (31)                  218
+                0x15, 0x00,                    #  Logical Minimum (0)                220
+                0x26, 0xff, 0x00,              #  Logical Maximum (255)              222
+                0x09, 0x42,                    #  Usage (Vendor Usage 0x42)          225
+                0x81, 0x00,                    #  Input (Data,Arr,Abs)               227
+                0x09, 0x42,                    #  Usage (Vendor Usage 0x42)          229
+                0x91, 0x00,                    #  Output (Data,Arr,Abs)              231
+                0xc0,                          # End Collection                      233
+            ]
+        ]
+
+        # HID++ 2.0 specific settings
+        self.feature_table = [
+            HIDPP20Features.IRoot,
+            HIDPP20Features.IFeatureSet,
+            HIDPP20Features.IFeatureInfo,
+            HIDPP20Features.DeviceNameAndType,
+            0x1d4b,
+            0x0020,
+            0x1001,
+            0x8070,
+            0x1300,
+            0x8100,
+            0x8110,
+            0x8060,
+            0x2201,
+            0x1802,
+            0x1803,
+            0x1805,
+            0x1806,
+            0x1811,
+            0x1830,
+            0x1890,
+            0x1891,
+            0x18a1,
+            0x1801,
+            0x18b1,
+            0x1df3,
+            0x1e00,
+            0x1eb0,
+            0x1863,
+            0x1e22
+        ]
```

Comment:
Not sure if github just stripped the file, but you want to call `super().__init__()` here.

Reply:
I forgot.